### PR TITLE
Supercharge demo and docs

### PR DESCRIPTION
**Pull request by ryantmer**
Just wanted to be able to more easily test things without having to use them in the LMS; also means we can make sure our components are robust in a more isolated fashion, so that we can break them out with confidence if needed.

### CODE REVIEW

File: demo/d2l-touch-menu-demo.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+						background-image="actions:close"
+						selection-handler="xHandler"
+						text="The X">
+					</d2l-touch-menu-item>
+				</d2l-touch-menu>
+			</div>
+			<script>
+				// This is some trickery required for the demo. In normal usage,
+				// this would not be required.
+				var menuParent = document.getElementById('menu-parent');
```
Comment (ryantmer) [inline demo/d2l-touch-menu-demo.html]: Just want to bring attention to this - I'm thinking we should update the touch-menu/touch-menu-item to not require the parent be a Polymer component, and instead just have the `selectionHandler` property be any function. Thoughts?

Comment (apalaniuk) [inline demo/d2l-touch-menu-demo.html]: That makes sense, and it's how I initially designed it (so long as you wanted to bind the selection handler imperatively).

File: README.md [hunk 2]
```diff
@@ -51,38 +114,36 @@ to link to the local d2l-my-courses-ui project
 
 5) Build and run brightspace-integration (will have to be rebuilt on any changes to d2l-my-courses-ui)
 
-Note: On Windows, there exists an issue with relative paths, which will prevent web-component-shards from completing successfully without modifying vulcanize to not use path.posix. See: https://github.com/Polymer/vulcanize/issues/338
+Note: On Windows, there exists an issue with relative paths, which will
```
Comment (apalaniuk) [inline README.md]: This actually doesn't seem to be an issue now, but I added it, so I can correct it in another PR =)

**Comment by apalaniuk** [comment]
LGTM, so long as you merge before #45 as that will change this documentation - then I'll update the docs in that PR.

File: README.md (hunk 2 above)
Comment (ryantmer) [inline README.md]: Ah, no worries, might as well remove it here

**Pull request merged**